homework.md

```diff
@@ -4,77 +4,78 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+white
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12,240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+HyperText Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+because its providing easy control over layout and presentation of website page by separating content from design
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language for the web
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1 they make collaboration very easy with others doc
+2 help change some code
+3 track information and help you to save your history
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+it take a repository from someone account to my account
 
-## Question 10 
+## Question 10
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+It to produce a copies of something
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+mean to ask for change committed to their own record to be considered
```

Comment:
I think you ask your project to be reviewed. Please correct me if I am wrong.

Reply:
I would say it is a method of submitting contributions to an open development project.